packages/teams-js/src/public/pages.ts

```diff
@@ -178,8 +190,12 @@ export namespace pages {
    * effect and will be a no-op when called.
    * @param navigateForward - Determines the direction to focus in host.
```

Comment:
New parameters need to be added to the comments so that they get turned into public documentation.

Reply:
addressed